crates/cubecl-core/src/codegen/integrator.rs

```diff
@@ -198,7 +207,19 @@ impl KernelSettings {
     /// Set kernel name.
     #[allow(dead_code)]
     pub fn kernel_name<S: AsRef<str>>(mut self, name: S) -> Self {
-        self.kernel_name = name.as_ref().to_string();
+        self.options.kernel_name = name.as_ref().to_string();
+        self
+    }
+
+    /// Activate debug symbols
+    pub fn debug_symbols(mut self) -> Self {
+        self.options.debug_symbols = true;
+        self
+    }
+
+    /// Set FP math mode
+    pub fn fp_math_mode(mut self, mode: FPMathMode) -> Self {
```

Comment:
`fp_math_mode` is an unclear name, and `FPMathMode` doesn’t follow the Rust naming conventions. We could remove FP entirely and keep only `MathMode`.

Reply:
I can agree on the name being bad, but I don't think `MathMode` is very descriptive either. Maybe just call it `FastMath`, since it maps to that in CUDA and it's more descriptive?